system-int-test/src/cmr/system_int_test/data2/umm_spec_service.clj

```diff
@@ -11,12 +11,21 @@
 
 (def ^:private sample-umm-service
   {:Name "AIRX3STD"
-   :Type "OPeNDAP"
+   :LongName "OPeNDAP Service for AIRS Level-3 retrieval products"
+   :Type "WEB SERVICES"
```

Comment:
Shouldn't the Type still be "OPeNDAP"? I know this is just a sample, but with that LongName and URL and OPeNDAP being in the list of enums it seems like that's the right Type to use.

Reply:
No, that will actually fail validation. The type is now one of a specific set of enums.

Reply:
Right I saw there was an enum, but OPeNDAP is in that list. If it's not in the list there is something wrong with our error message.
```
(str "/Type instance value (\"\") not found in enum "
                    "(possible values: [\"OPeNDAP\",\"THREDDS\",\"WEB SERVICES\","
                    "\"WCS\",\"WMS\",\"SOFTWARE PACKAGE\",\"TOOL\",\"WEB PORTAL\","
                    "\"International Web Portal\",\"MODEL\",\"NOT PROVIDED\"])")```

Reply:
You are too right! I had missed that when I looked at the schema changes and the (non-OPeNDAP) example files from Simon. My statement about the failure of validation is incorrect!

I will change back to OPeNDAP ...